completions/find

```diff
@@ -36,7 +36,7 @@ _comp_cmd_find()
             return
             ;;
         -group)
-            COMPREPLY=($(compgen -g -- "$cur" 2>/dev/null))
+            _comp_compgen -- -g 2>/dev/null
```

Comment:
Not introduced in this PR, but I wonder what the reason for `2>/dev/null` is here.
```suggestion
            _comp_compgen -- -g
```

Reply:
I'm unsure about it too.

I've identified the commit ad561ac1c.

```
commit ad561ac1cabc23bc477bddc5f0abb8af92701925
Author: ianmacd <>
Date:   Sun Apr 14 22:41:04 2002 +0000

    - avoid errors when comp{gen,lete} -g don't work on an unpatched bash 2.05
```

So this was a workaround for bash-2.05. We can clean it up now.